py/torch_tensorrt/fx/test/converters/aten_op/test_cat_aten.py

```diff
@@ -53,6 +53,41 @@ def forward(self, x, y):
             expected_ops={torch.ops.aten.cat.default},
         )
 
+    def test_cat_no_dim(self):
+        class Cat(nn.Module):
+            def forward(self, x, y, z):
+                return torch.cat((x, y, z))
+
+        inputs = [torch.randn(2, 1, 3), torch.randn(1, 1, 3), torch.randn(3, 1, 3)]
+        self.run_test(
+            Cat(),
+            inputs,
+            expected_ops={torch.ops.aten.cat.default},
+        )
+
+    def test_cat_dynamic_shape_no_dim(self):
```

Comment:
Yes, the regular implementation works when the dimension is negative now, and I've re-enabled the tests for that. The new case I added does not need separate tests for negative dim, since it is exclusively for the no-dim-specified case, where the user calls `torch.cat` with the default dimension (0).